virt-v2v/cold/entrypoint.go

```diff
@@ -299,3 +296,67 @@ func isValidSource(source string) bool {
 		return false
 	}
 }
+
+func addFirmwareToYaml(filePath string) (err error) {
```

Comment:
Since virt-v2v currently set in a different workspace I was not able to use the yaml package so did a simple file write instead.

Reply:
yeah, another reason to move it to the main workspace - but we'll do that as a separate change later on